clinica/iotools/utils/data_handling.py

```diff
@@ -235,6 +241,24 @@ def create_merge_file(
         cprint("End of CAPS information merge.", lvl="debug")
 
 
+def create_participants_df(bids_dir: str) -> pandas.DataFrame:
+    """
+    Function to create participant.tsv "on the fly" in case it is not found in the BIDS directory
+    """
+    import pandas as pd
+    import os
+
+    subjects_ids = [
+        name
+        for name in os.listdir(bids_dir)
+        if os.path.isdir(os.path.join(bids_dir, name)) and "sub" in name
+    ]
```

Comment:
It might be possible to simplify this part with `Path.glob("sub-*")`?

Reply:
not relevant anymore as I removed the function